### PR TITLE
makes the pdf templates source the stylescheet from the webapp

### PR DESCRIPTION
**Pull request by esotrope**
Goes along with gitlab change: https://gitlab.com/fairwaytech/sbac-config-repo/merge_requests/59
Will merge when config repo is ready.

### CODE REVIEW

File: pdf-report-processor/src/main/resources/application.yml [hunk 1]
```diff
@@ -67,3 +67,4 @@ app:
     options:
       dpi: 276
       page-size: Letter
+      user-style-sheet: http://localhost:8080/assets/css/sbac-ui-kit-wkhtmltopdf.min.css
```
Comment (esotrope) [inline pdf-report-processor/src/main/resources/application.yml]: with this solution the docker wkhtmltopdf service will be hitting the webapp service to get the pdf stylesheet.
the only alternative is embedding the css into the html template or creating our own custom dockerized wkhtmltopdf service rest interface that has the stylesheet.

Thoughts?

Comment (malaffoon) [inline pdf-report-processor/src/main/resources/application.yml]: Seems like a good approach but perhaps a parking-lot brainstorming discussion?

Comment (bgsavage) [inline pdf-report-processor/src/main/resources/application.yml]: Is there a way to use relative pathing here?

Comment (esotrope) [inline pdf-report-processor/src/main/resources/application.yml]: if you did the host:port would resolve to the wkhtmltopdf service's (localhost:8082)

**Pull request merged**